src/titlepane/styles/titlePane.css

```diff
@@ -0,0 +1,34 @@
+@import '../../common/styles/variables.css';
+
+.main {
+	overflow: hidden;
+	box-sizing: border-box;
+}
+
+.main * {
+	box-sizing: border-box;
+}
+
+.title {
+	background-color: var(--title-bar-bg);
```

Comment:
Probably need to change some of these now that @tomdye has added an initial theme. Also, why does this file exist alongside `titlePane.m.css`?